ofApp.cpp

```diff
@@ -1,387 +0,0 @@
-    if (screen == screens::home) {
-        backgroundImage.draw(0, 0, 1200, 750); // this should cover the whole home screen at all times
-        leftObj.landlockedLife = '1'; // resets number of lives back to 1 - change this to whatever number you want if you want more lives but 1 is enough i reckon
-        font.drawString(wrapString(titleText, 1100, true), 50, 100); // there are 2 calls here being made. one is to wrap the string, the other is to take the string which is now wrapped and display it on the GUI
-        font2.drawString(wrappedTextTeaCup, 50, 200);
-        
-        choice1.set(200, 400, 250, 175); // where you see 4 numbers like this they generally represent xPos, yPos, width and height relative to (0,0), which is the top left corner
-        choice2.set(700, 400, 250, 175);
-        
-        ofDrawRectangle(choice1);
-        ofDrawRectangle(choice2);
-
-        populationButton.draw(200, 400, 250, 175);
-        areaButton.draw(700, 400, 250, 175);
-    }
-    else if (screen == screens::game) {
-        backgroundImage.draw(0, 0, 1200, 750); // this should cover the whole game screen at all times
-
-        leftOptionImage.draw(0, 0, 600, 750);
-        font.drawString(wrapString(leftObj.jsonStringName, 550, true), 50, 65);
-        //ofDrawBitmapString(leftObj.population, 20, 500); //YzxNkP99T9WYb8
-        //ofDrawBitmapString(leftObj.area, 20, 525); //YzxNkP99T9WYb8
-        leftObj.jsonImg.draw(0, 130, 600, 300); //this does not work since not all flags are the same shape - for example switzerland's flag is square not rectangular
-
-        rightOptionImage.draw(600, 0, 600, 750);
-        font.drawString(wrapString(rightObj.jsonStringName, 550, true), 650, 65);
-        //ofDrawBitmapString(rightObj.population, 620, 500); //YzxNkP99T9WYb8
-        //ofDrawBitmapString(rightObj.area, 620, 525); //YzxNkP99T9WYb8
-        rightObj.jsonImg.draw(600, 130, 600, 300); 
-
-
-        leftOptionButton.set(0, 0, 600, 750);
-        rightOptionButton.set(600, 0, 600, 750);
-
-        // score drawn last as it is going to cover both sides of the screen and this has a first come first served basis for drawing. whatever is drawn first ends up on the bottom. and by bottom i OF COURSE mean farthest away from the camera on the z-axis
-        font.drawString("Score:", 450, 550); // will not need to wrap the score since ppl unlikely to get a score worthy of being wrapped
-        font.drawString(ofToString(leftObj.score), 640, 550); // need ofToString to convert it quickly to a string so it is valid - found that here - https://openframeworks.cc/documentation/graphics/ofGraphics/#show_ofToString
-
-        ofSetColor(208, 72, 72);
-
-        if (leftObj.colour != "left") { // if the mouse is hovering over your option, then redraw the name of said option in a different colour - thats all this is doing but it provides a nice affect
```

Comment:
Wouldn't it be more readable to check if the colour attribute is equal to a value instead of !=

Also, you are checking the colour property and using "left" as an evaluation? Not sure this makes sense